server/app.py

```diff
@@ -1,8 +1,32 @@
-from flask import Flask
+from flask import Flask, jsonify, request
+from flask_cors import CORS
```

Comment:
Whenever you install a new Python package, please run `pip freeze > requirements.txt`  to ensure the package list is updated.
Otherwise, people may not be able to run your version locally as the package is not installed locally.